product_docs/docs/pgd/5/data_migration/edbloader.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: EDB*Loader and PGD
+navTitle: EDB*Loader
+description: EDB*Loader is a high-speed data loading utility for EDB Postgres Advanced Server.
+---
+
+[EDB\*Loader](/epas/latest/database_administration/02_edb_loader/) is a high-speed data loading utility for EDB Postgres Advanced Server (EPAS). It provides an interface compatible with Oracle databases, allowing you to load data into EDB Postgres Advanced Server. It's designed to load large volumes of data into EDB Postgres Advanced Server quickly and efficiently.
+
+The EDB\*Loader command line utility loads data from an input source, which can be a file flat file, pipe, or other programs, into one or more tables using a subset of the parameters offered by Oracle SQL\*Loader. 
+
+## Use with PGD
+ 
+As EDB\*Loader is a utility for EDB Postgres Advanced Server, it's available for EDB Postgres Distributed when EDB Postgres Advanced Server is the database in use for PGD data nodes. PGD deployments can use EDB\*Loader in the same way as it's used on EDB Postgres Advanced Server. Refer to the [EDB\*Loader documentation](/epas/latest/database_administration/02_edb_loader/) for more information on how to use EDB\*Loader with EDB Postgres Advanced Server.
+
+### Replication and EDB\*Loader
+
+As with EDB Postgres Advanced Server, EDB\*Loader works with PGD in a replication environment. But, unlike EPAS with physical replication, it isn't possible to use the [direct path load method](/epas/latest/database_administration/02_edb_loader/invoking_edb_loader/direct_path_load/) to load data into the replica nodes. Only the node connected to by EDB\*Loader gets the data that's EDB\*Loader is loading because the direct path load method skips use of the WAL, upon which logical replication relies. 
```

Comment:
This is still wrong :)

The direct path load method does not work with replication, period, regardless of physical or PGD or anything else.

The difference is that since all PGD nodes are writeable, one can do direct path on all of them and have the data everywhere.

With physical replication, direct path load only loads data to primary, the replicas will not have it (yes it's weird).